tap_s3_csv/sync.py

```diff
@@ -162,6 +171,22 @@ def handle_file(config, s3_path, table_spec, stream, extension, file_handler=Non
     return 0
 
 
+def get_cols_from_metadata(stream):
+    try:
+        mdata = metadata.to_map(stream['metadata'])
+        # Get keys of mdata that are tuples, filter out empty key, and get list of second elements from tuples
+        cols_from_mdata = [key[1]
```

Comment:
this is equivalent to column order?

Reply:
yes, col order was custom added for data preprocess ticket, but was not backwards compatible change.